src/components/Footer/index.js

```diff
@@ -225,26 +240,28 @@ const StyledFooter = styled.footer`
           }
         }
         .compliance2 {
-           display: flex; 
-           justify-content: center;
-           align-items: center;
-           img {
-             max-width: 55px;
-             display: inline-block;
-             &:not(:last-child) {
-               margin-right: 1rem;
-             }
-             &.bestchange {
-               max-height: 40px;
-             }
-             &.kurs {
+          display: inline-block;
+          
+          justify-content: center;
+          align-items: center;
+          
+          img {
+            max-width: 60px;
+            display: inline-block;
+            margin-right: 2rem;
+            &:not(:last-child) {
```

Comment:
@chiragneb @mrtrader1313 
why not use simple bootstrap classes instead of complex css selectors?